src/goto-programs/goto_program_template.h

```diff
@@ -240,6 +241,13 @@ class goto_program_templatet
 
       return false;
     }
+
+    irep_idt to_string() const
```

Comment:
I think this should be returning a std::string - I don't think we need to de-duplicate the storage on this one (or have very efficient comparison).